verl/trainer/config/ppo_megatron_trainer.yaml

```diff
@@ -164,7 +164,12 @@ actor_rollout_ref:
       enable: False  # should set rollout.name to sglang_async if True
       max_turns: null  # null for no limit (default max_length // 3)
       tool_config_path: null  # null for no tool
-      format: chatml  # chatml, more formats will be supported in the future
+      format: chatml  # deprecated, no effect anymore
```

Comment:
Could we remove this unnecessary option to avoid misleading users into thinking it offers deprecated support instead of being fully removed?